book/index.md

```diff
@@ -39,6 +39,55 @@ API resources in the second row (PipelineRun, Snapshot) should primarilly be tho
 data-plane resources. The system responds to user requests by creating and managing the lifecycle of
 these resources.
 
+- An [Application] represents a functionally coherent set of git branches that should be built,
+  tested, and released together. The user provides and names their [Applications]. They are
+  generally long-lived and don't change much after they are created.
+- A [Component] represents an image to build from a particular git branch (and a particular context
+  directory). It is owned by an [Application]. The user provides and names their [Components] and
+  specifies which git repositories and branches they relate to. The user may add, remove, or change
```

Comment:
I wouldn't consider Components really _relating to_ git repos/branches. Instead, I think of it more of which repos/branches define how the underlying artifact is built.

Reply:
Agreed. Updated!